nilearn/maskers/tests/test_surface_labels_masker.py

```diff
@@ -392,10 +393,6 @@ def test_surface_label_masker_inverse_transform_list_surf_images(
     assert img.shape == (surf_label_img.mesh.n_vertices, 7)
 
 
-def test_surface_label_masker_list_inverse_transform_output(surf_mesh):
```

Comment:
unrelated: seems like I started defining a test in a previous branch but then refactored other tests to test the same thing